### DISCUSSION TITLE
On Android devices, content at the bottom of the screen is concealed when the keyboard is opened, the same thing occurs in the hotel app

### DISCUSSION THREAD

**Discussion by gaurikalp** [EnsembleUI/ensemble#119] [General]
On Android devices, content at the bottom of the screen is concealed when the keyboard is opened, the same thing occurs in the hotel app . Please find the screen shot , Login button and QR Code id hidden

![WhatsApp Image 2022-11-23 at 5 25 34 PM](https://user-images.githubusercontent.com/115991454/204260264-8f282548-bda5-404a-8084-9e4127dc4db0.jpeg)

**Comment by amin-nas** [comment]
@gaurikalp what is your expected behavior? can you share a link to this screen?

**Reply by gaurikalp** [reply]
There is a login button and QR Code immediately below the password entry, but they are hidden because the keyboard appears , Does the keyboard have extra margin?  At the very least, the login button should be visible whether the keyboard is open or not.

https://studio.ensembleui.com/app/e38acc8b-e00f-4c92-a992-1191cc372df8/screen/ONdJ1eqb20w2Y50wer4s

**Comment by vusters** [comment]
@gaurikalp the latest Android version addressed this issue.

**Reply by gaurikalp** [reply]
Yes Problem fixed in updated version. Thank you.
